spec/services/token_manager_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'rails_helper'
+
+RSpec.describe TokenManager do
+  subject{ TokenManager }
+  let(:user){ create(:user) }
+
+  describe ".generate_token" do
+    let(:token){ subject.generate_token(user.id) }
+
+    it 'generates a valid token' do
+      expect(token.split('.').count).to eq 3
+      expect{subject.decode(token)}.not_to raise_error
+    end
+
+    describe 'token expiration' do
+      it 'sets expiration time to 24hrs by default' do
+        decoded = subject.decode(token)
+        expect(decoded.first['exp']).to be_within(100).of(24.hours.from_now.to_i)
```

Comment:
yeah sure, but the `be_within(100).of(24.hours.from_now.to_i)` doesn't seem so clear, as my first of many questions was what is `100`? and then if someone is thinking of the `within_` you would have expected a range, which isn't, as we're passing an integer in here